shared/middleware/helpers/storage_id.rb

```diff
@@ -8,7 +8,7 @@ def create_storage_id_cookie
   response.set_cookie(storage_id_cookie_name, {
     value:CGI.escape(storage_encrypt_id(storage_id)),
     domain:".#{site}",
-    path:'/v3/apps',
+    path:'/',
```

Comment:
Definitely don't do this, this will break Varnish caching completely. I don't actually think there will be a problem with /editcode/5 because even though that page doesn't send the cookie (because it's not on the cookie path), when that page makes a request to a /v3/apps path, the cookie is passed.